fvm/meter/meter_test.go

```diff
@@ -671,19 +670,11 @@ func TestWeightedComputationMetering(t *testing.T) {
 
 func TestMemoryWeights(t *testing.T) {
 	for kind := common.MemoryKindUnknown + 1; kind < common.MemoryKindLast; kind++ {
-		weight, ok := meter.DefaultMemoryWeights[kind]
-		if !assert.True(t, ok, fmt.Sprintf("missing weight for memory kind '%s'", kind.String())) {
-			continue
-		}
-		assert.Greater(
```

Comment:
Don't we need to keep checking whether the assigned weight is non-zero?

Reply:
Good point! I had just applied the diff from #8068, where this change was needed because the added weights needed to be zero. Reverting